### ISSUE TITLE
Color output

### ISSUE THREAD

**Issue by arrowd** [nick8325/quickcheck#307]
It would be neat, if QuickCheck's messages like

```
+++ OK, passed 1 test.
```

or

```
*** Failed! Falsified (after 1 test):  
```

be colored.

**Comment by phadej** [comment]
The test runners like `tasty` or `hspec` do that, don't they?

**Comment by arrowd** [comment]
Hum. I'm running tests with `stack test`, but I'll try out `tasty`/`hspec`, thanks.

**Comment by tom-audm** [comment]
`microspec` is pretty close to simply being QuickCheck + color output

**Comment by Bodigrim** [comment]
I agree with @phadej, let's leave coloring to testing frameworks. `QuickCheck` should not make guesses whether an output handle supports any colors.
